public/content/sv/welcome/links-documentation.md

```diff
@@ -0,0 +1,17 @@
+# Länkar och dokumentation
+
+Efter att ha avslutat rundturen, kan du kolla in några av de följande länkarna för mer information:
+
+**NOTERA:** Följande länkar är på engelska
+
+* [Lär dig D, för nybörjare](http://ddili.org/ders/d.en/index.html)
+* [Lär dig D, för erfarna programmerare](http://wiki.dlang.org/Coming_From) från andra språk
+* [D handledning](https://wiki.dlang.org/Tutorials)
+* [Överblick av Ds unika funktioner](http://dlang.org/overview.html)
+* [Vanliga frågor om D](http://dlang.org/faq.html)
+
+### Hjälp
+
+* Fråga frågor på [#d](irc://irc.freenode.net/d) IRC kanalen på freenode ([webb gränssnitt](https://kiwiirc.com/client/irc.freenode.net/d))
+* Få hjälp på [D.learn](http://forum.dlang.org/group/learn)
+* Öppna en 'issue' [i D bugg trackern](https://issues.dlang.org)
```

Comment:
Something I forgot to comment on before is if `bugg trackern` the best word for the issue tracker.
The closes thing I can find is `ärendehantering system`, but this doesn't really tell you it is about bugs.